integration-tests/tests.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+
+import subprocess, argparse, shlex, sys, re, os
+
+run_options = ['put','get','exist']
+
+test_run_config = dict()
+iter = 0
+try:
+    for item in run_options:
+        test_run_config[item]=str(20000+(iter % int(os.environ['BOB_NODES_AMOUNT']))) #used in get_run_args()
```

Comment:
This is the only place in this script, where env variable is used. This is a bit strange. I suggest to pass `min_port` and `max_port` as command line arguments and used them to generate target ports

Reply:
fixed